src/lib.rs

```diff
@@ -97,13 +98,13 @@
 //!
 //! # Details
 //!
-//! Birli reads visibilities with [`MWALib`] and uses CXX to bind to the [`AOFlagger`] C++ library.
-//! For more details about AOFlagger's interface, check out the [`aoflagger::AOFlagger`]
-//! documentation
+//! Birli reads visibilities with [`MWALib`] and uses CXX to bind to the
+//! [`AOFlagger`] C++ library. For more details about AOFlagger's interface,
+//! check out the [`aoflagger::AOFlagger`] documentation
 //!
-//! [`MWALib`]: https://github.com/MWATelescope/mwalib
-//! [`AOFlagger`]: https://gitlab.com/aroffringa/aoflagger
-//! [`aoflagger::AOFlagger`]: http://www.andreoffringa.org/aoflagger/doxygen/classaoflagger_1_1AOFlagger.html
+//! [`MWALib`]: https://github.com/MWATelescope/mwalib [`AOFlagger`]:
```

Comment:
yup, looks like the formatter can't handle rustdoc links https://doc.rust-lang.org/rustdoc/linking-to-items-by-name.html